data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
+    "description": "Gun in a cane.  A custom-built .44 Magnum rifle carefully integrated into a walking cane.  It's also reinforced for hand-to-hand combat.  Never underestimate a handicapped person.",
+    "weight": 950,
+    "volume": 4,
+    "price": 400000,
+    "bashing": 15,
+    "to_hit": 2,
+    "material": [ "steel", "aluminum", "wood" ],
+    "symbol": "/",
+    "color": "dark_gray",
+    "skill": "rifle",
+    "ammo": "44",
+    "ranged_damage": 11,
+    "dispersion": 100,
+    "sight_dispersion": 75,
+    "aim_speed": 7,
+    "recoil": 90,
+    "durability": 8,
+    "loudness_modifier": 25,
+    "clip_size": 1,
+    "reload": 120,
+    "barrel_length": 11,  "//": "It has a 40in. barrel.",
+    "valid_mod_locations": [ [ "accessories", 1 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 1 ], [ "muzzle", 1 ], [ "sights", 1 ] ],
```

Comment:
The design is modular. The gun features detachable furniture that can be swapped for grips if the user wants. The rubber tip around the barrel can be removed to expose an attachment point capable of mounting a suppressor.